src/api-reference/expense/expense-report/v4.workflows.markdown

```diff
@@ -101,64 +121,222 @@ PATCH operations in Expense Reports v4 conform to the JSON Merge Patch specifica
 
-#### <a name="patch-approve-sendback-recall-report-request-schema"></a>Sample Requests (short and long URL)
+This API supports both company level and user level access tokens.
+
+### Scopes
+
+`expense.report.read` - Refer to [Scope Usage](#scope-usage) for full details.
+
+`expense.report.readwrite` - Refer to [Scope Usage](#scope-usage) for full details.
+
+### Parameters
+
+| Name       | Type     | Format | Description                                                                                                                                         |
+|------------|----------|--------|-----------------------------------------------------------------------------------------------------------------------------------------------------|
+| `userID`   | `string` | -      | **Required** The unique identifier of the SAP Concur user. Use [Identity v4 API](/api-reference/profile/v4.identity.html) to retrieve the `userID`. |
+| `reportId` | `string` | -      | **Required** The unique identifier of the report that is being read.                                                                                |
+
+### Headers
+
+* [RFC 7231 Accept-Language](https://tools.ietf.org/html/rfc7231#section-5.3.5)
+* [RFC 7231 Content-Type](https://tools.ietf.org/html/rfc7231#section-3.1.1.5)
+* [RFC 7231 Content-Encoding](https://tools.ietf.org/html/rfc7231#section-3.1.2.2)
+* [RFC 7234 Cache-Control](https://tools.ietf.org/html/rfc7234#section-5.2)
+* [RFC 7232 If-Modified-Since](https://tools.ietf.org/html/rfc7232#section-3.3)
+* [RFC 7231 Accept-Encoding](https://tools.ietf.org/html/rfc7231#section-5.3.4)
+* [RFC 7235 Authorization](https://tools.ietf.org/html/rfc7235#section-4.2) - Bearer Token that identifies the caller.
+  This is the Company or User access token.
+* `Concur-CorrelationId` is a Concur specific custom header used for technical support in the form of
+  a [RFC 4122 A Universally Unique IDentifier (UUID) URN Namespace](https://tools.ietf.org/html/rfc4122)
+
+### Request
+
+#### URI Template
 
 ```shell
-curl --location --request PATCH 'https://us.api.concursolutions.com/expensereports/reports/764428DD6A664AF0BFCB/approve' \
---header 'Authorization: Bearer {access_token}' 
---header 'Content-Type: application/json' 
---data-raw '{
-"comment":"Approved"
-}'
+https://{datacenterURI}/expensereports/v4/users/{userId}/reports/{reportId}/costObjectsForApprover
+```
 
-curl --location --request PATCH 'https://us.api.concursolutions.com/expensereports/v4/users/32C2FCC3-B2E8-4907-9672-5B3F49B1C643/context/MANAGER/reports/764428DD6A664AF0BFCB/approve' \
---header 'Authorization: Bearer {access_token}' 
+#### Curl Example
+
+```shell
+curl --location --request GET https://integration.api.concursolutions.com/expensereports/v4/users/aa024d31-95eb-4270-b9yz-cb6b5f558a7c/reports/97F923D6935D4426BF9C/costObjectsForApprover
+--header 'Authorization: Bearer {access_token}' \
+--header 'Concur-CorrelationId: abcd-efgh-xyzw' \
```

Comment:
We are not exposing `Concur-CorrelationId`. It is considered as internal only param. Remove.